.devcontainer/devcontainer.json

```diff
@@ -25,7 +25,7 @@
       "version": "latest"
     },
     "ghcr.io/devcontainers/features/node:1": {
-      "version": "latest"
+      "version": "20.16.0"
```

Comment:
This looks as if it is pinned at the patch, but that little `:1` in line 27 means use latest major version, see [devcontainer feature usage](https://github.com/devcontainers/features#usage).

Reply:
interesting